PubnativeLite/PubnativeLite/VAST/PNLiteVASTPlayerInterstitialViewController.m

```diff
@@ -71,7 +71,14 @@ - (void)loadFullScreenPlayerWithPresenter:(HyBidInterstitialPresenter *)intersti
     self.adModel = ad;
     self.player = [[PNLiteVASTPlayerViewController alloc] initPlayerWithContentInfo:self.adModel.contentInfo isInterstital:YES];
     self.player.delegate = self;
-    [self.player loadWithVastString:self.adModel.vast];
+    
+//    VAST Verification Test Script
```

Comment:
probably we can remove this commented out code

Reply:
Won't push it to production of course but i think it'll be good to keep it until we have a valid response from MockDSP. 👍